frontends/bsd/src/screens/write_in_adjudication_screen.tsx

```diff
@@ -45,6 +45,9 @@ import { Prose } from '../components/prose';
 import { Screen } from '../components/screen';
 import { AppContext } from '../contexts/app_context';
 
+type AnyWriteInAdjudicationReasonInfo =
+  | WriteInAdjudicationReasonInfo
+  | UnmarkedWriteInAdjudicationReasonInfo;
```

Comment:
How about this? I think if I were using these types I might assume accidentally that `WriteInAdjudicationReasonInfo` is the main type

```suggestion
type WriteInAdjudicationReasonInfo =
  | MarkedWriteInAdjudicationReasonInfo
  | UnmarkedWriteInAdjudicationReasonInfo;
```

Reply:
I changed this to make a broader global change like you suggested.